### PR TITLE
Filter activities in progress page by category

### PR DESCRIPTION
**Pull request by rachelchenrc**
Add the activity category filter to the progress page.

Screen Shot 2020-01-06 at 4 30 59 PM

[Trello Card](https://trello.com/c/eZEIpsFY/28-15-allow-jobseekers-to-filter-the-list-view-of-all-activities-history-page-by-activity-category-bucket)

[Live Env](https://nj-career-network-dev4.firebaseapp.com/progress/)

### CODE REVIEW

File: components/history/Filter.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  },
+}));
+
+export default function Filter(props) {
+  const classes = useStyles();
+  const { filterOptions, onChange } = props;
+
+  return (
+    <div>
+      <FormControl>
```
Comment (hannahtwobulls) [inline components/history/Filter.js]: I don't believe the `<div>` is necessary

Comment (rachelchenrc) [reply components/history/Filter.js]: removed

**Comment by rossdakin** [comment]
@endely also viewable here with the latest changes as well: https://nj-career-network-dev1.firebaseapp.com/

**Comment by rachelchenrc** [comment]
Dev 4 should show the filter now. Sorry.

**Comment by rachelchenrc** [comment]
Updated! Thanks @endely

**Pull request merged**